### PR TITLE
packet-injector: add support for UDP packets

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

File: contrib/vagrant/setup-common.sh [hunk 1]
```diff
@@ -3,3 +3,4 @@
 sudo dnf -y install python2
 sudo setenforce 0
 sudo sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/sysconfig/selinux
+[ `hostname` == "agent1" ] && sleep 180 || true
```
Comment (nplanel) [inline contrib/vagrant/setup-common.sh]: debug ?

Comment (lebauce) [reply contrib/vagrant/setup-common.sh]: Ooops. Thanks

**Comment by lebauce** [comment]
run skydive-scale-tests

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-elasticsearch

**Pull request merged**